packages/stark-build/config/webpack.config.js

```diff
@@ -172,7 +172,9 @@ module.exports = (config, options) => {
 			? {
 					devServer: {
 						// See: https://webpack.js.org/configuration/dev-server/#devserverwritetodisk-
-						writeToDisk: true,
+						devMiddleware: {
```

Comment:
Could you adapt the documentation URL ?

```suggestion
// See: https://webpack.js.org/configuration/dev-server/#devserverdevmiddleware
devMiddleware: {
	writeToDisk: true
},
```

Reply:
done